<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovimientosPedido</class>
 <widget class="QDialog" name="MovimientosPedido">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pedido de productos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nº Pedido</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtNPedido">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtCentro">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Centro pedido</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnServir">
         <property name="text">
          <string>Servir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAniadirProd">
         <property name="text">
          <string>Añadir productos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAniadirKit">
         <property name="text">
          <string>Añadir kit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEliminarProd">
         <property name="text">
          <string>Eliminar productos</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" rowspan="2" colspan="3">
      <widget class="QTableWidget" name="tableDetalles">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="ptedtComentarios">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>495</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>495</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Comentarios</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImprimir">
       <property name="text">
        <string>Imprimir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnServir</tabstop>
  <tabstop>btnAniadirProd</tabstop>
  <tabstop>btnAniadirKit</tabstop>
  <tabstop>btnEliminarProd</tabstop>
  <tabstop>edtCentro</tabstop>
  <tabstop>edtNPedido</tabstop>
  <tabstop>tableDetalles</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MovimientosPedido</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MovimientosPedido</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
